app/views/subscriptions/index.html.erb

```diff
@@ -10,10 +10,15 @@
       <p><%= page_entries_info @subscriptions, entry_name: 'Subscription' %></p>
     <% end %>
 
+    <p>
+      <em><span class="fa fa-info-circle"></span> Subscription operations may take some time. If you are not seeing what you expect below, please <%= link_to 'refresh the page', 'javascript:history.go(0)', class: 'eui-btn--link link-lift' %>.</em>
```

Comment:
if we are doing the 'try 20 times', do we still need this?

Reply:
Spoke with PO, she wanted a new message in its place.